data/json/recipes/other/tool.json

```diff
@@ -128,5 +128,29 @@
       [ [ "pot_copper", 1 ] ],
       [ [ "hose", 2 ] ]
     ]
+  }, 
+  {
+    "result": "food_processor",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 4 ],
```

Comment:
I'm not sure about this, but if this recipe requires only 4 level of fabrication, then this line (`"skills_required": [ "fabrication", 4 ],`) is not needed?
By the way, we can add requirement of some level of electronics, as now we are using electric motor and electric scrap in the recipe.